### DISCUSSION TITLE
HMS-1000-2T startet andauernd neu

### DISCUSSION THREAD

**Discussion by neolauren-cmd** [hoylabs/OpenDTU-OnBattery#1043] [General]
Hallo, ich beobachte seit Tage das mein WR alle paar Minuten neustartet. Dies geschieht nur, wenn AC seitig keine Abnahme ist. Ich kann aber DC seitig keine Spannungsschwankungen erkennen. Hab den WR auch schon mal vom Netz genommen, dann ist 10 Minuten Ruhe, danach dieselbe Geschichte. Liegt hier ein  Defekt vom WR vor oder was kann ich noch machen?

PS: Es erscheint kein rotes Kreuz im WebIf vor dem WR.

Danke und Gruß 
Neo

**Comment by schlimmchen** [comment]
Wie hoch ist dein "lower power limit"?

**Reply by neolauren-cmd** [reply]
70 Watt. Aber AC seitig ist ausgeschaltet. Und trotzdem startet der WR immer wieder neu. Gefühlt ist nur 1 Sekunde die DC Spannung weg.

**Comment by Manos1966** [comment]
> AC seitig ist ausgeschaltet

Ist es nicht der Moment wo die Sicherungsausschaltung des Wechselrichters prueft ob AC-Seitig 230V existieren und FALLS NICHT den Wechselrichter STOPPT?
Wie oft prüft der Wechselrichter ob AC-Seitig 230V existieren, damit er mit der Produktion anfangen kann? (ich glaube, die Antwort auf dieser Frage hast du bereits gefunden) 😜

**Reply by neolauren-cmd** [reply]
Aber der WR startet ca. alle 10 Sekunden neu mit der Melde ID 1. Ich habe noch 3 weitere Hoymiles, da ist das nicht. Alle hängen an einen Solarflow von Zendure.
Bei dem anderen WR kommt nur die Meldung Netztrennung, wenn AC seitig ausgesteckt ist, ansonsten bleiben die alle im Standby. 
D.h. DC Seitig liegt immer Spannung an, ich starte alle manuell 1x täglich. Es könnte doch mal jemand testen, wenn 230 V AC ausgesteckt ist , ob der WR immer neustartet.

**Reply by Manos1966** [reply]
Alle das gleiche Modell (HMS-1000-2T)?

(Ich kann morgen testen wie die HM Serie sich verhält, aber ich habe keine HMS)

**Reply by neolauren-cmd** [reply]
2x HM und 2x HMS
Problem ist erst seit ca. 1 Woche. Vorher war alles okay und der WR ist nur neugestartet, wenn ich es wollte.

**Reply by Manos1966** [reply]
Gerade getestet,

HM1200, nur Netztrennung (einmal)
![Netztrennung](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/b00a339e-9e05-4297-91b8-8545ddc35b13)

wenn du aber sagst, von zwei HMS (beide am Solarflow?) nur der eine hat dieses Verhalten, dann befürchte ich es liegt am Wechselrichter.

Falls aber nur einer HMS am Solarflow angeschlossen ist und der andere nicht! Dann haette ich vermutet es koennte an irgendwelche intelligente Steuerung der Solarflow liegen, die regelmäßig die DC Stromversorgung testet und dadurch den HMS zum Neustart zwingt.

**Reply by neolauren-cmd** [reply]
Beide am Solarflow. Und Problem erst seit einer Woche.

Ist es für den WR eigentlich schädlich, ihn AC seitig abzuschalten, obwohl DC seitig Spannung und Leistung anliegt?

**Reply by Manos1966** [reply]
Ich habe nie von Problemen auf der AC-Seite gehoert.

Ich befuerchte, die Fehlersuche wird Schmerthaft:
1. Ohne OpenDTU-OnBattery Steuerung - Wie verhaelt er sich?
2. Ohne Anschluss an die Batterie -  Wie verhaelt er sich?

PS. Vorsicht beim Anschliessen / Trennen der Eingaenge an die Batterie (falls Solarflow kein SOFT-START Option hat), ich hoffe du weiss darueber Bescheid

**Comment by neolauren-cmd** [comment]
Hallo Zusammen,

leider ist der Fehler doch wieder gekommen, auch mit dem neuen WR. Fehler tritt auf, wenn ich nicht am Solarflow bin, sondern auch direkt an den Platten.

Meine Vermutung ist, dass es an der DTU liegt.

Ich hatte diese mal 20 Minuten weg, dann war der Fehler nicht. Er ist erst gekommen, seitedem ich sie wieder angesteckt habe. Teste gerade mal die letzten Softwarestände durch, ob mir auffällt mit welcher Version es auftritt.

Ich gebe wieder Bescheid.

Gruß 
Neo

**Comment by neolauren-cmd** [comment]
Hallo Zusammen,

ich konnte jetzt ausführlich testen. Es liegt eindeutig an der DTU Firmware. Ich habe jetzt die vom 23.03.24 drauf. Da hab ich keine Neustarts mehr. Erst mit den letzten 2 habe ich diese Probleme. 
Soll ich das noch irgendwo melden?

Gruß 
Neo

**Reply by schlimmchen** [reply]
Hm. Kannst du denn einmal mitloggen (serielle Konsole, oder von mir aus Web-Konsole), wenn die OpenDTU-OnBattery den WR neustartet? Du hast den DPL an, korrekt? Und der WR, der neugestartet wird obwohl du es nicht erwartest, der wird vom DPL gesteuert?

Hm... Wenn das Ding AC-seitig getrennt ist, dann wird niemals `isProducing()` wahr, vermute ich. Wenn aber der DPL entschieden hat, der WR solle laufen, dann startet er den WR irgendwann neu, wenn der einfach nicht den Ziel-Zustand erreicht.

Ist denn das Feld im Live View gelb, während AC-seitig getrennt ist?

Hier könnte man nachsteuern, indem man das Einschalten des WR nicht versucht, wenn dieser meldet, dass AC-seitig gar keine Spannung anliegt. Find ich nicht so geil, denn ich verstehe den Grund nicht, warum die AC-seitig trennen musst, und unterstelle, dass du das bitte einfach lassen solltest, weil der Anwendungsfall für OpenDTU-OnBattery jener ist, dass der WR AC-seitig auch angeschlossen ist und produzieren könnte, wenn er denn vom DPL dazu aufgefordert wird.

**Comment by neolauren-cmd** [comment]
Hallo, 

ja der WR wird vom DPL gesteuert. Dieser ist dann gelb. Neustart ist ca. alle 2 Minuten, wenn AC getrennt.
AC trenne ich, da ddrxWR am Solarflow hängt und dieser dann 100 Prozent hat. Ich schalte dann AC seitig aus, da ich noch einen zweiten Solarflow mit anderer Ausrichtung habe und diesen WR dann AC seitig einschalte. Dieser ist aber nicht am DPL. Komischweise funktionieren alle Firmware bis 23.03 2024. Erst danach wurde irgendwas geändert.

Ich schau mal, wie ich mitloggen kann.

**Comment by schlimmchen** [comment]
> Komischweise funktionieren alle Firmware bis 23.03 2024.

Das ist weder komisch noch merkwürdig, das Feature kam erst danach rein, siehe #901.

**Reply by neolauren-cmd** [reply]
Ah okay. Da erklärt das dann wohl. Dann bleib ich bei der Firmware vom 23.03.

Die Frage ist halt, ob das schädlich für den WR ist, wenn ich ihn AC seitg abschalte und der DPL trotzdem arbeitet. Bisher kontte ich nichts feststellen  und das System läuft seit einem Jahr so.
